tests/TestHelpers/GraphHelper.php

```diff
@@ -44,7 +44,7 @@ public static function isUUIDv4(string $id): int {
 	 */
 	private static function getFullUrl(string $baseUrl, string $path): string {
 		$fullUrl = $baseUrl;
-		if (\substr($fullUrl, -1) !== '/') {
+		if (!str_ends_with($fullUrl, '/')) {
```

Comment:
Note: `str_ends_with` is a new PHP 8 function. This code is only going to be used to test "ocis-style" back-ends, so we should be able to run the test pipelines with PHP 8.* 

Developers who run tests locally will need to use PHP 8. They might also be doing work on ownCloud core, which needs PHP 7.4. So they will need to have multiple PHP versions installed on their development machines, and be able to switch between them. I don't think that it should be an issue.

Reply:
should I revert this @phil-davis this may create confusions.